src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -980,10 +986,11 @@ public String getReleaseNotesForFixVersion(String projectKey, String versionName
      * @param projectKey The project key
      * @param toVersion  The new fixVersion
      * @param query      A JQL Query
+     * @param build 
```

Comment:
Would be nice to have a short description for this param so that it doesn't stand out from the others ;-)